src/MOI_wrapper.jl

```diff
@@ -216,4 +218,92 @@ function backward!(model::Optimizer, params::Array{String}, dl_dz::Array{Float64
         end
     end
     return grads
-end
\ No newline at end of file
+end
+
+# MOI supports
+
+function MOI.supports(::Optimizer,
+    ::MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}})
+    return true
+end
+
+function MOI.supports(::Optimizer,
+    ::MOI.ObjectiveFunction{MOI.ScalarQuadraticFunction{Float64}})
+    return true
+end
+
+function MOI.supports(::Optimizer, ::MOI.AbstractModelAttribute)
```

Comment:
Because of this, all ObjectiveFunction will be supported. You should mirror the signatures of the `set` functions somehow, e.g.
```julia
function MOI.supports(::Optimizer, ::MOI.AbstractModelAttribute)
    return true
end
function MOI.supports(::Optimizer, ::MOI.ObjectiveFunction)
    return false
end
function MOI.supports(::Optimizer, ::MOI.ObjectiveFunction{F}) where F <: ...
    return true
end
```

Reply:
ok. edited!